src/lib/SkyraClient.ts

```diff
@@ -1,38 +1,34 @@
 /* eslint-disable @typescript-eslint/no-invalid-this */
 // Import all dependencies
```

Comment:
Either split the imports in categories again, or remove all these comments if we're not splitting the imports anymore.

Personally I'm more in favour of the latter option.